ux_research.md

```diff
@@ -0,0 +1,89 @@
+
+To identify a design's most important usability problems, [testing 5 unique users](https://www.nngroup.com/articles/why-you-only-need-to-test-with-5-users/) is typically enough. It's a better use of resources to run many small tests and revise the design between each one so we can fix the usability flaws as we identify them, rather than scheduling one or two big expensive studies each year. Iterative design is the best way to increase the quality of user experience. The more versions and interface ideas you test with users, the better. Finally, listening to what people say is often misleading: you have to watch what they actually do.
+
+### Recruiting and Screening Research Candidates
+
+Although there are many ways to recruit test candidates, at DLS we have two available to us, maintaining a pool of volunteers and "hallway recruiting". No matter which method is selected for a project, make sure to disclose that we intend to record the test session before scheduling. If a prospective participant is not comfortable with being recorded, it would be best to find another participant.
+
+#### Pool of existing participants
+
+It makes sense for us to recruit from our existing users and staff to establish a pool of volunteers who are willing to either try out new features or participate in research opportunities in general. One challenge our team faces is that we are not able to compensate these testers for their labor. To remedy this problem, we can look to the large pool of staff and student workers who are presently compensated for the tasks they are asked to perform at the Library, and perhaps even collaborate to establish a network of potential test candidates at the University. We will need the permission from PUL leadership to pursue this method of recruitment. Using a pool of existing participants is best for more formal testing of a larger set of tasks.
+
+There is also the cost of creating a pool of participants and maintaining it, as well as doing the work of scheduling and coordinating participants. There might also be some sampling and confirmation bias due to participants’ existing knowledge of certain systems, so we may have to do some simple screening of candidates to counter this.
+
+The results of these tests should generally be private, so we don’t need to perform an IRB review.
+
+#### Intercept studies (or "hallway recruiting")
+
+Intercept studies are useful for getting quick feedback on a feature or interface. A group of 5-10 users should be sufficient for this type of testing. The basic idea is simply to approach people in the building (or for remote folks, reach out to University contacts via Slack) and ask “Would you be willing to take 5 minutes to participate in a quick survey?”
+
+These types of studies can be done virtually (via Zoom) or in person (in the lobby, office hallway, or coffee shop). They are ideal for quick feedback on a small subset of features.
+
+### Scripting Task Scenarios
```

Comment:
should we add something in this section about how to identify good tasks? for example, we recently interviewed librarians. and/or maybe a reference to the section on FAQs below?

Reply:
I think perhaps this is done in the addition of the checklist?

Reply:
I think https://github.com/pulibrary/dls-handbook/issues/44 probably covers what I meant.